CONTRIBUTING.md

```diff
@@ -68,10 +68,11 @@ When adding a new file to a subfolder of content (for example, `handbook/tools`)
 This repo offers some `package.json` scripts to help you out:
 
 ```sh
-npm install      # installs our standard Markdown linter and site builder
-npm run lint     # runs the linter to check for style errors
-npm run hooks    # installs the git commit hook that runs the linter before you commit
-npm run serve    # runs the website locally
+npm install             # installs our standard Markdown linter and site builder
+npm run lint            # runs the linter to check for style errors
+npm run hooks           # installs the git commit hook that runs the linter before you commit
+npm run sc <filepath>   # runs markdown spell checker on <filepath>
```

Comment:
```suggestion
```

I think it might be best if:

* `npm run spellcheck` ran spellchecks across all relevant files (`handbook/**/*.md`, `resources/**/*.md`)
* `npm run lint` also spellchecks (which gets it included in CI/CD, and gives contributors one less thing to run separately)

Reply:
see https://github.com/ubclaunchpad/docs/pull/125#issuecomment-647993802